README.md

```diff
@@ -53,7 +53,9 @@ the build is reproducible.
 image for testing Pull Requests built with the open build service. This needs to be copied manually.
 - http://minima-mirror.mgr.prv.suse.net/distribution/leap/: oss and non-oss repositories.
 
-## Used image versions in the CI test suite
+## CI Test suite
```

Comment:
```suggestion
## CI Testsuite
```

Reply:
Nope. We use `test suite` for quite some time now 😉